src/mlpack/core/data/load_numeric_csv.hpp

```diff
@@ -14,8 +14,32 @@
 
 #include "load_csv.hpp"
 
-namespace mlpack{
-namespace data{
+namespace mlpack {
+namespace data {
+
+/**
+ * A safe function to get negative or positive infinity, which avoids unary
+ * minus on an unsigned type.  This works around a Visual Studio bug.
+ * (TODO: add a link?)
```

Comment:
Is this something you want to do before merging this?

Reply:
Thanks for pointing that out... I had forgotten to handle that.  After digging around a bit more, it's not an error, it's just a warning, and I don't have a relevant link for it.  I fixed my comment in 45394e01c6. :+1: